README.md

```diff
@@ -249,7 +249,41 @@ Returns an array of Routes, one for each destination.
 | boundingBox       | {  maxLat: Double,  maxLong: Double,  minLat: Double,  minLong: Double  } | The most extreme points of the route. Used to center the map on client-side (with some padding) |
 | numberOfTransfers | Int                                                                               | Number of transfers in route. Default 0.    
 
+----------
+# **/search** • POST
+
+**Description**: Returns a list of bus stops and google autocomplete search results given a query string.
+
+## Parameters
+
+*required* **query** : String - “st”
```

Comment:
I don't think you need the "st" (Look at the docs for like `/tracking`)/ The param is straight forward unlike lat, lng coords so it's fine to just put String